SPECS/libtiff/libtiff.spec

```diff
@@ -67,6 +68,10 @@ make %{?_smp_mflags} -k check
 %{_mandir}/man3/*
 
 %changelog
+
```

Comment:
Nit-pick clean-up comment:
```suggestion
```

Reply:
I Added a new entry in the change log with no space inbetween, marking this one as resolved.